produce/lib/produce/itunes_connect.rb

```diff
@@ -51,9 +53,13 @@ def create_new_app
         UI.crash!("Something went wrong when creating the new app - it's not listed in the App's list") unless application
 
         UI.message("Ensuring version number")
-        application.ensure_version!(Produce.config[:app_version], platform: Produce.config[:platform]) if Produce.config[:app_version]
+        platforms.each do |platform|
+          application.ensure_version!(Produce.config[:app_version], platform: platform) if Produce.config[:app_version]
+        end
 
         UI.success("Successfully created new app '#{Produce.config[:app_name]}' on App Store Connect with ID #{application.apple_id}")
+
+        return Spaceship::Tunes::Application.find(@full_bundle_identifier, mac: platforms.first == "osx").apple_id
```

Comment:
this seems redundant, no?

Reply:
Ahhh yeah yeah, I can find a way to improve 😊